.github/workflows/python.yml

```diff
@@ -70,11 +70,11 @@ jobs:
             title: AMD64 Conda Python 3.6 Pandas 0.23
             python: 3.6
             pandas: 0.23
-          - name: conda-python-3.6-pandas-latest
-            cache: conda-python-3.6
+          - name: conda-python-3.7-pandas-latest
+            cache: conda-python-3.7
             image: conda-python-pandas
-            title: AMD64 Conda Python 3.6 Pandas latest
-            python: 3.6
+            title: AMD64 Conda Python 3.7 Pandas latest
+            python: 3.7
```

Comment:
@kszucs is it fine to simply change this like the above? Or does something else need to be updated as well? 

CI for this build seems to pass, but the "Python / docker" one fails

(I am changing "pandas-latest" to use 3.7, as right now we have many builds using 3.6, and almost none using 3.7, eg the pandas 0.23 above also uses python 3.6)

Reply:
SGTM